pom.xml

```diff
@@ -359,7 +359,7 @@
 			<dependency>
 				<groupId>dom4j</groupId>
 				<artifactId>dom4j</artifactId>
-				<version>1.6.1</version>
+				<version>20040902.021138</version>
```

Comment:
do we really have this version under dom4j,kindly check well on the maven repository for dom4j!!

Reply:
` <version>1.6.1</version>` this seems to be the latest full release still

Reply:
@mozzy11  true. So what are we updating here? Kindly help me

Reply:
you can close the PR , no update needed